src/js/components/form-elements/TableBodyVirtualized.jsx

```diff
@@ -28,18 +28,20 @@ class TableBodyVirtualized extends Component {
 
   getHeight() {
     const { fieldsConfig: { subfieldKey, getDynamicRowAttr }, fields, properties } = this.props;
-    const { totalCount } = properties;
     let height = 0;
     const maxTableHeight = window.innerHeight < 900 ?
       // 0.35 * window.innerHeight = 35vh from table-content class in StockMovement.scss
       0.35 * window.innerHeight : 0.40 * window.innerHeight;
 
     if (!subfieldKey) {
-      if (totalCount * 28 > maxTableHeight) {
-        height = maxTableHeight;
-      } else if (totalCount > 0) {
-        height = totalCount * 28;
-      }
+      height = fields.value.reduce((acc, field) => {
+        const dynamicAttr = getDynamicRowAttr ?
+          getDynamicRowAttr({ ...properties, rowValues: field }) : {};
+        if (dynamicAttr.hideRow || height > maxTableHeight) {
+          return acc;
+        }
+        return acc + 28;
```

Comment:
this `28` value can be seen in many places in the file, I think it would be a good idea to maybe move it to a named variable. It will be much easier if we ever want to change this value.
Also a name for the variable should provide an explanation to what this value stands for, because at the moment I don't know what `28` is _(margin, height, padding ?)_

Reply:
As you can see, `height =`, so obviously it is height.
Sure I can move it to a variable, as I said in the comment, I hadn't wanted to make much refactor to this, since anyway it was a pain in the neck to understand what is going on here, so I didn't want to cause any regression in case I don't understand any case.
But of course moving this value to a variable isn't "risky", so I'll do that, thanks for the advice.